proxy/src/console_redirect_proxy.rs

```diff
@@ -35,6 +35,7 @@ pub async fn task_main(
     socket2::SockRef::from(&listener).set_keepalive(true)?;
 
     let connections = tokio_util::task::task_tracker::TaskTracker::new();
+    let cancellations = Arc::new(tokio_util::task::task_tracker::TaskTracker::new());
```

Comment:
TaskTracker internally uses an Arc, so we don't need another one here

Reply:
right! Fixed